templates/pipelines/pipeline_builder.html

```diff
@@ -2,6 +2,8 @@
 {% load static %}
 {% block page_head %}
   <link rel="stylesheet" href="{% static 'css/pipeline.css' %}">
+  <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/6.5.1/css/all.min.css" integrity="sha512-..." crossorigin="anonymous" referrerpolicy="no-referrer" />
```

Comment:
No need to include this here. We have these modules included through the webpack bundler